runtime/testy/runtimeUtil.py

```diff
@@ -9,6 +9,7 @@ class BAD_EVENTS(Enum):
   STUDENT_CODE_TIMEOUT    = "Student Code Timed Out"
   UNKNOWN_PROCESS         = "Unknown State Manager process name"
   STATE_MANAGER_KEY_ERROR = "Error accessing key in State Manager"
+  END_EVENT               = "Process terminated" # Used for testing
```

Comment:
description doesn't really signal that it ended gracefully. I would change the message